### PR TITLE
feat: dockerize test suite

### PR DESCRIPTION
**Pull request by jawid-h**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# for easier app bind mounting for local development
+WORKDIR /
+
+COPY package.json package-lock.json ./
+RUN npm ci --production
+
+FROM node:12-alpine
+
+LABEL maintainer="Dash Developers <dev@dash.org>"
+LABEL description="Test suite Node.JS"
```
Comment (shumkov) [inline Dockerfile:20]: Test suite for Dash Platform

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+FROM node:12-alpine
+
+RUN apk update && \
+    apk --no-cache upgrade && \
+    apk add --no-cache git \
+                       openssh-client \
```
Comment (shumkov) [inline Dockerfile:6]: do you think we need it?

Comment (Alex-Werner) [reply Dockerfile:6]: It's useful if we work with private repo. 
Probably not as we pushed everything public and this repo is public too.

Comment (shumkov) [reply Dockerfile:6]: Oh, right. But why do we need python? To build some npm dependency?

File: .travis.yml [hunk 3]
```diff
@@ -30,3 +32,10 @@ script:
   - ~/.local/bin/dash-network test $DEVNET
   - cd $TRAVIS_BUILD_DIR
   - npm run test
+
+deploy:
+  provider: script
+  script: bash bin/travis-deploy.sh
+  on:
+    tags: true
+    repo: dashevo/dash-platform-test-suite
```
Comment (shumkov) [inline .travis.yml:41]: `platform-test-suite`

**Comment by shumkov** [comment]
Looks good, but we need to merge #27 first.

File: bin/run-test.sh [hunk 4]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
```
Comment (shumkov) [inline bin/run-test.sh:1]: ?

**Pull request merged**